apps/condo/domains/organization/schema/OrganizationEmployeeRole.test.js

```diff
@@ -0,0 +1,244 @@
+            const admin = await makeLoggedInAdminClient()
+            const [organization] = await createTestOrganization(admin)
+            const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                canManageRoles: false,
+            })
+            const managerUserClient = await makeClientWithNewRegisteredAndLoggedInUser()
+            await createTestOrganizationEmployee(admin, organization, managerUserClient.user, role)
+
+            let thrownError
+            try {
+                await updateTestOrganizationEmployeeRole(managerUserClient, role.id)
+            } catch (e) {
+                thrownError = e
+            }
+            expect(thrownError).toBeDefined()
+            expect(thrownError.errors[0]).toMatchObject({
+                'message': 'You do not have access to this resource',
+                'name': 'AccessDeniedError',
+                'path': ['obj'],
+            })
+            expect(thrownError.data).toEqual({ 'obj': null })
+        })
+
+    })
+
+    test('anonymous: update OrganizationEmployeeRole', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const [organization] = await createTestOrganization(admin)
+        const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+            canManageRoles: false,
+        })
+        const managerUserClient = await makeClientWithNewRegisteredAndLoggedInUser()
+        await createTestOrganizationEmployee(admin, organization, managerUserClient.user, role)
+
+        const client = await makeClient()
+
+        let thrownError
+        try {
+            await updateTestOrganizationEmployeeRole(client, role.id)
+        } catch (e) {
+            thrownError = e
+        }
+        expect(thrownError).toBeDefined()
+        expect(thrownError.errors[0]).toMatchObject({
+            'message': 'You do not have access to this resource',
+            'name': 'AccessDeniedError',
+            'path': ['obj'],
+        })
+        expect(thrownError.data).toEqual({ 'obj': null })
+    })
+
+    test('user: delete OrganizationEmployeeRole', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const [organization] = await createTestOrganization(admin)
+        const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+            canManageRoles: true,
+        })
+        const managerUserClient = await makeClientWithNewRegisteredAndLoggedInUser()
+        await createTestOrganizationEmployee(admin, organization, managerUserClient.user, role)
+
+        let thrownError
+        try {
+            await OrganizationEmployeeRole.delete(managerUserClient, role.id)
+        } catch (e) {
+            thrownError = e
+        }
+        expect(thrownError).toBeDefined()
+        expect(thrownError.errors[0].message).toMatch('Cannot query field "deleteOrganizationEmployeeRole" on type "Mutation"')
```

Comment:
why not `toMatchObject` ?

Reply:
Method `toMatchObject` is not suitable here, because error message contains suggestions, generated by Keystone:
> Cannot query field "deleteOrganizationEmployeeRole" on type "Mutation". Did you mean "createOrganizationEmployeeRole", "deleteOrganizationEmployee", "deleteOrganizationEmployees", "updateOrganizationEmployeeRole", or "createOrganizationEmployeeRoles"?
We don't know this suggestions list in advance.
Added this comment in `18d1625d`.